src/Network/NWProtocolFramerOptions.cs

```diff
@@ -0,0 +1,64 @@
+//
+// NSProtocolFramerOptions : Bindings the Netowrk nw_protocol_options API focus on Framer options.
+//
+// Authors:
+//   Manuel de la Pena <mandel@microsoft.com>
+//
+// Copyrigh 2019 Microsoft Inc
```

Comment:
```suggestion
// Copyright 2019 Microsoft Inc
```
also maybe 2022? I'm not sure how that works tbh